### PR TITLE
add yt link to code ep6

### PR DESCRIPTION
**Pull request by paulczar**
Signed-off-by: Paul Czarkowski <username.taken@gmail.com>

### CODE REVIEW

File: content/tv/code/0006.md [hunk 1]
```diff
@@ -8,16 +8,12 @@ minutes: 120
 aliases: ["/tv/code/6"] #redirect alias
 episode: "6"
 # Run this after the show is uploaded to youtube
```
Comment (tiffanyfay) [inline content/tv/code/0006.md:10]: Why delete the comments? It's easier to copy paste to new ones with it. Also, you forgot a comment.

Comment (paulczar) [reply content/tv/code/0006.md]: We don't use the youtube image for code, we keep it at the episode image. I was cleaning up to match the template ( https://raw.githubusercontent.com/vmware-tanzu/tanzu-dev-portal/main/content/tv/code/_template.md )

**Review by tiffanyfay — APPROVED** [review]

File: content/tv/code/0006.md [hunk 2]
```diff
@@ -8,16 +8,12 @@ minutes: 120
...
-# wget -O static/images/tv/episodes/code/6.jpg https://img.youtube.com/vi/TODO/mqdefault.jpg
-
-# change all images to `/images/tv/episodes/code/name-of-guest.png` when created.
-# change episode_image to `/images/tv/episodes/code/6.jpg` when YT uploaded.
 episode_image: "/images/tv/episodes/code/mario-gray.png"
```
[the remark this answers is not in the document]
Comment (paulczar) [reply content/tv/code/0006.md:10]: We're actually re-using the author's episode image each time ... so its always going to be their name.   We're not running the wget for code.

[the remark this answers is not in the document]
Comment (tiffanyfay) [reply content/tv/code/0006.md:10]: Uhhh I didn't post this an hour ago. I couldn't post at all...

**Review by tiffanyfay — APPROVED** [review]

**Pull request merged**